Homework/homework05

ki8as5 opened the pull request.

ki8as5 committed:
add homework01

ki8as5 committed:
add homework02

ki8as5 committed:
add homework03

ki8as5 committed:
add homework04

Allyonzy commented on src/ru/innopolis/java/homework01/Main.java:
```diff
@@ -0,0 +1,5 @@
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("3 4 6 8");
```
Можно вынести в переменные числа и распечатать их через printf

Allyonzy commented on src/ru/innopolis/java/homework03/TV.java:
```diff
@@ -0,0 +1,74 @@
+    private String Color;
+    private String Model;
+    Scanner go = new Scanner(System.in);
+
```
Добавить пустой конструктор - без чтения со Scanner

Allyonzy commented on src/ru/innopolis/java/homework03/TV.java:
```diff
@@ -0,0 +1,74 @@
+
+public class TV {
+    private String Brand ;
+    private double Diagonal;
```
Переменные (поля) класса называются с маленькой группы в camelCase

Allyonzy commented on src/ru/innopolis/java/homework03/TV.java:
```diff
@@ -0,0 +1,74 @@
+    private double Diagonal;
+    private String Color;
+    private String Model;
+    Scanner go = new Scanner(System.in);
```
Scanner вызывается в специальном классе или Main (App). Отделить чтение данных от класса

Allyonzy commented on src/ru/innopolis/java/homework03/TV.java:
```diff
@@ -0,0 +1,74 @@
+    public void setModel(String model) {
+        Model = model;
+    }
+    public String toString(){
```
Не хватает @Override

Allyonzy reviewed (approved):
Принято. Учесть рекомендации и смержить с main самостоятельно.

ki8as5 committed:
Создал переменные и вывел через принт ф

ki8as5 committed:
убрал сканнер, вывел параметрами 2 способами через сеттер и конструктор

ki8as5 committed:
добавил Override

ki8as5 committed:
сделал дополнительную задачу в 5 домашней работе, добавил 3 новых поля, создал конструкторы, переопределил toString,equals,hasCode.
В классе app создал массив записал туда параметры телевизоров и про поможи цикла for вывел только включенные телевизоры с допустимым уровнем звука

ki8as5 committed:
добавил пакет
